$NetBSD: patch-tmake_lib_macosx-c++_tmake.conf,v 1.1.2.2 2013/08/21 12:51:24 tron Exp $

Drop an option that is unsupported by the default compiler in Mac OS 10.8.
--- tmake/lib/macosx-c++/tmake.conf.orig	2013-08-20 17:30:43.000000000 +0000
+++ tmake/lib/macosx-c++/tmake.conf
@@ -9,7 +9,7 @@ CONFIG			= qt warn_on release
 
 TMAKE_CC		= cc
 TMAKE_CFLAGS		= -pipe 
-TMAKE_CFLAGS_WARN_ON	= -Wall -W -Wno-deprecated-declarations -Wno-invalid-source-encoding
+TMAKE_CFLAGS_WARN_ON	= -Wall -W -Wno-deprecated-declarations
 TMAKE_CFLAGS_WARN_OFF	=
 TMAKE_CFLAGS_RELEASE	= -O2
 TMAKE_CFLAGS_DEBUG	= -g -fstack-protector
